### DISCUSSION TITLE
How to change first day of the week

### DISCUSSION THREAD

**Discussion by optimistic5** [firefly-iii/firefly-iii#7454] [Firefly III]
I would like to change first day of the week from Sunday to Monday.
How I can do it?
![image](https://user-images.githubusercontent.com/25054050/235459968-1d99432d-6a78-4253-82db-c764d0abb6de.png)

**Comment by JC5** [comment]
I'm fairly sure this is your browser locale that decides this.

**Reply by optimistic5** [reply]
Can't find such setting in Firefox.
